pkgs/sops-install-secrets/nixos-test.nix

```diff
@@ -14,10 +14,17 @@ let
         secrets."nested/test/file".owner = "example-user";
       };
 
-      users.users.example-user = {
-        isNormalUser = true;
-        hashedPasswordFile = config.sops.secrets.test_key.path;
-      };
+      users.users.example-user = lib.mkMerge [
+        (lib.mkIf (! config.systemd.sysusers.enable) {
+          isNormalUser = true;
+          hashedPasswordFile = config.sops.secrets.test_key.path;
+        })
+        (lib.mkIf config.systemd.sysusers.enable {
+          isSystemUser = true;
+          group = "users";
+          hashedPasswordFile = config.sops.secrets.test_key.path;
+        })
```

Comment:
sysusers can no longer be used to create normal users, there was an assertion added. Only system users are supported now.

Reply:
Ok.